sql/moz-fx-data-shared-prod/firefox_ios/ltv_states/view.sql

```diff
@@ -0,0 +1,58 @@
+{% set max_weeks = 32 %}
+{% set death_time = 160 %}
+{% set lookback = 28 %}
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.ltv_states`
+AS
+WITH base_layer AS (
+  SELECT
+    client_id,
+    sample_id,
+    submission_date,
+    first_seen_date,
+    days_since_first_seen,
+    days_since_seen,
+    BIT_COUNT(
+      `mozfun`.bytes.extract_bits(days_seen_bytes, - {{lookback}}, {{lookback}})
+    ) AS pattern,
+    IF(
+      (durations > 0)
+      AND (BIT_COUNT(`mozfun`.bytes.extract_bits(days_seen_bytes, -1, 1)) = 1),
+      1,
+      0
+    ) AS active,
+    ad_clicks,
+    adjust_network,
+    first_reported_country,
+    first_reported_isp,
+    {{ death_time }} AS death_time,
+    {{ max_weeks }} AS max_weeks
+  FROM
+    `moz-fx-data-shared-prod.firefox_ios_derived.ltv_states_v1`
+)
+SELECT
+  client_id,
+  sample_id,
+  submission_date,
+  first_seen_date,
+  days_since_first_seen,
+  days_since_seen,
+  pattern,
+  active,
+  ad_clicks,
+  adjust_network,
+  first_reported_country,
+  first_reported_isp,
+  death_time,
+  max_weeks,
+  mozfun.ltv.get_state_ios_v2(
```

Comment:
WDYT about doing this the way we did with Android? i.e. explicitly naming states based on the state function, and accommodating multiple  https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/fenix/ltv_states/view.sql#L21-L22